### PR TITLE
🎨 Frontend: Usage table: show ``root_parent_project_name`` instead of ``project_name``

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

Show ``root_parent_project_name`` instead of ``project_name`` (which are the same if the project has no parent)

## Related issue/s

follows https://github.com/ITISFoundation/osparc-simcore/pull/5891

## Dev-ops checklist

- [ ] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by matusdrobuliak66 — APPROVED** [review]
:+1:

File: services/static-webserver/client/source/class/osparc/desktop/credits/UsageTableModel.js [hunk 1]
```diff
@@ -71,15 +50,30 @@ qx.Class.define("osparc.desktop.credits.UsageTableModel", {
...
   },
 
+  statics: {
+    SERVER_MAX_LIMIT: 49,
+    COLUMN_ID_TO_DB_COLUMN_MAP: {
+      0: "root_parent_project_name",
+      1: "node_name",
+      2: "service_key",
+      3: "started_at",
+      // 4: DONT SORT BY DURATION
```
Comment (GitHK) [inline services/static-webserver/client/source/class/osparc/desktop/credits/UsageTableModel.js:60]: Please check that below is not an error.

This looks suspicious. Why is there a `// 4:` also I would expect the index to also include a 4, you don't have one

Comment (GitHK) [reply services/static-webserver/client/source/class/osparc/desktop/credits/UsageTableModel.js:60]: if you don't need 4 I would suggest something like
```suggestion
      // 4: (not used) SORTING BY DURATION
```

**Review by GitHK — APPROVED** [review]
👍

**Review by pcrespov — COMMENTED** [review]
:+1:

**Review by sanderegg — APPROVED** [review]

**Review by ignapas — APPROVED** [review]
lgtm

**Pull request merged**